content/en/docs/components/pipelines/sdk/connect-api.md

```diff
@@ -4,227 +4,337 @@ description = "How to connect to Kubeflow Pipelines using the SDK client and con
+This guide describes how to:
+* [Connect to Kubeflow Pipelines from outside your cluster](#connect-to-kubeflow-pipelines-from-outside-your-cluster)
+* [Connect to Kubeflow Pipelines from within the same cluster](#connect-to-kubeflow-pipelines-from-within-the-same-cluster)
+* [Configure SDK client by environment variables](#configure-sdk-client-by-environment-variables)
 
+## Before you begin
 
-The Kubeflow Pipelines REST API is available at the same endpoint as the Kubeflow Pipelines user interface (UI).
-The SDK client can send requests to this endpoint to upload pipelines, create pipeline runs, schedule recurring runs, and more.
-
+* Deploy Kubeflow Pipelines using one of the [installation options](/docs/components/pipelines/installation/overview/)
+* [Install the Kubeflow Pipelines SDK](/docs/components/pipelines/sdk/install-sdk/) python library
 
-## Before you begin
 
-* You need a Kubeflow Pipelines deployment using one of the [installation options](/docs/components/pipelines/installation/overview/).
-* [Install the Kubeflow Pipelines SDK](/docs/components/pipelines/sdk/install-sdk/).
+---
 
 
 ## Connect to Kubeflow Pipelines from outside your cluster
 
-Kubeflow distributions secure the Kubeflow Pipelines public endpoint with authentication and authorization.
-Since Kubeflow distributions can have different authentication and authorization requirements, the steps needed to connect to your Kubeflow Pipelines instance might be different depending on the Kubeflow distribution you installed. Refer to documentation for [your Kubeflow distribution](/docs/started/installing-kubeflow/):
-
-* [Connecting to Kubeflow Pipelines on Google Cloud using the SDK](/docs/distributions/gke/pipelines/authentication-sdk/)
-* [Kubeflow Pipelines on AWS](https://www.kubeflow.org/docs/distributions/aws/pipeline/#authenticate-kubeflow-pipeline-using-sdk-inside-cluster)
-* [Authentication using OIDC in Azure](/docs/distributions/azure/authentication-oidc/)
-* [Pipelines on IBM Cloud Kubernetes Service (IKS)](/docs/distributions/ibm/pipelines/)
+### Non-multi-user mode
 
-For [Kubeflow Pipelines standalone](https://www.kubeflow.org/docs/components/pipelines/installation/standalone-deployment/) and [Google Cloud AI Platform Pipelines](/docs/components/pipelines/installation/overview/#google-cloud-ai-platform-pipelines), you can also connect to the API via `kubectl port-forward`.
+For Kubeflow Pipelines in non-multi-user mode, you can expose the Pipelines API locally by using `kubectl port-forward`.
 
-Kubeflow Pipelines standalone deploys a Kubernetes service named `ml-pipeline-ui` in your Kubernetes cluster without extra authentication.
+Kubeflow Pipelines (standalone) deploys an unsecured Kubernetes `Service` named `ml-pipeline-ui` which requires no authentication.
```

Comment:
Suggestion:

`Kubeflow Pipelines Standalone doesn't enforce authentication in `ml-pipeline-ui` service`.

Reply:
Changed.